ARCHITECTURE.md

```diff
@@ -60,16 +60,22 @@ Base | CRDs
 Traffic Management | Pilot
 Policy | Policy
 Telemetry | Telemetry
+Telemetry | Prometheus
+Telemetry | Prometheus Operator
+Telemetry | Grafana
+Telemetry | Kiali
+Telemetry | Tracing
```

Comment:
These need an asterisk as they are not core components (e.g. they are not in the proto). Maybe take them out of the table and include a note under it that mentions them?

Reply:
done.